clients/java/src/main/java/io/zeebe/client/impl/ZeebeClientCredentials.java

```diff
@@ -50,9 +51,29 @@ public String getTokenType() {
     return tokenType;
   }
 
+  @JsonSetter("expiry")
+  public void setExpiry(final String expiry) {
+    this.expiry = ZonedDateTime.parse(expiry);
+  }
+
+  @JsonSetter("expires_in")
+  public void setExpiresIn(final String expiresIn) {
+    this.expiry = ZonedDateTime.now().plusSeconds(Long.parseLong(expiresIn));
```

Comment:
Is it in the spec that expiresIn is passed a seconds? Where is expires in coming from?

Reply:
Yes, https://tools.ietf.org/html/rfc6749#section-5.1 . I'm not sure if this is what you mean by "where is it coming from" but it comes from the auth response, like the access token and the bearer type.